python/paddle/fluid/framework.py

```diff
@@ -2756,6 +2756,15 @@ class Program(object):
     it will contains nested block.
```

Comment:
contains->contain

Reply:
Changed, but these are not related to my PR